app/src/main/java/com/android/partagix/model/Database.kt

```diff
@@ -478,28 +478,29 @@ class Database(database: FirebaseFirestore = Firebase.firestore) {
    * Set a review for a loan, i.e a rank and an optional comment
    *
    * @param loanId the loan's id
-   * @param userId the reviewed user's id
+   * @param reviewedUserId the reviewed user's id
    * @param rank the rank to be set, must be between 0.5 and 5
    * @param comment an optional comment to be set
    */
   fun setReview(
       loanId: String,
-      userId: String,
+      reviewedUserId: String,
       rank: Double,
       comment: String = "",
   ) {
     getLoans { loans ->
       for (loan in loans) {
 
         if (loan.state == LoanState.FINISHED // only finished loans
-        && loan.id == loanId && loan.idLender == userId) {
+        && loan.id == loanId && loan.idLender == reviewedUserId) {
```

Comment:
This is not great for code readability and complexity, there might be a more elegant way to do it.